string-manipulation-1/src/get-last-char.js

```diff
@@ -0,0 +1,14 @@
+/* exported getLastChar */
+/*
+declar a new empty string variable
+use for loop to iterate the string
+in the for loop, assign the last index of the string array to the new empty string variable
+after the for loop return to the new string variable
+*/
+function getLastChar(string) {
+  var char = '';
+  for (var i = 0; i < string.length; i++) {
+    char = string[string.length - 1];
```

Comment:
Same thing here, you are only getting one character, there is no repeated action here that requires a loop

Reply:
you are right, i was thinking too hard